Automattic/jetpack
# [Package] Backup: Added backup warning UI

#### Changes proposed in this Pull Request:

* Adds support for backups with warnings
* If a backup completes with any warnings the cloud icon will display yellow and the normal cloud button will be replaced with `Backup is completed with some files missing. See your backup in the cloud for more details.` and a link out to Jetpack Cloud.

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Have a backup team member change a successful backup on your site to display a warning
* Ensure the cloud icon changes to yellow
* Ensure the 'See all your backups' button changes to text linking to Jetpack Cloud

<img width="1148" alt="Screen Shot 2022-06-08 at 3 54 47 PM" src="https://user-images.githubusercontent.com/1992658/172705438-ab1ce2ff-c8ca-4580-b2c0-7251b00964a4.png">
